mkdocs.yml

```diff
@@ -4,19 +4,20 @@ repo_url: https://github.com/aws/aws-application-networking-k8s
 edit_uri: edit/main/docs/
 strict: true
 
-
 nav:
   - Home: index.md
+  - Concepts:
+    - Introduction: concepts/concepts.md
+    - Concepts: concepts/overview.md
   - User Guides:
```

Comment:
Deleted the file because it created duplicity, this part is already explained in the walkthrough (multi-cluster part). (and it is not in the [current version](https://github.com/aws/aws-application-networking-k8s/blob/main/mkdocs.yml) neither)